Docs/user_guide/getting_started.md

```diff
@@ -123,6 +123,15 @@ The included libcrypto.so.1.1 in the Slicer installation is incompatible with th
 
     $SLICER_ROOT/lib/Slicer-4.xx/libcrypto.*
 
+If you double-clicked on the app and it doesn't launch, you can try this (in order):
+- verify execution rights of the app (Right click on the app > properties > executable as a program checked) or in terminal, in the app folder:
+    
+    chmod +x Slicer
+- launch the app from terminal :
+    
+    ./Slicer
+- if you get the "error: Failed to obtain launcher executable name !" message : move the app in your Software Home folder and double click on the app, it should launch.
```

Comment:
It would help if you could provide more context for this.  Is there something special on Fedora that files are downloaded by default to a place where they can't be executed?

Reply:
On fedora, only User's Home folders get execution rights. But for some reason, i do not know why but i experienced it : if you unpack the files in the download folder : you get the error message, if you unpack in the "Applications" folder, it just work fine... That's why i suggested the addition to the documentation.

Reply:
This also makes sense and would be nice to mention in the text.  Maybe some documentation about installing applications in these systems.